thrift/jaeger.thrift

```diff
@@ -77,6 +77,14 @@ struct Batch {
   2: required list<Span> spans
 }
 
+struct BatchSubmitResponse {
+    1: required bool ok
```

Comment:
I suggest adding a comment that that Agent is not expected to do anything with not-ok response, other than perhaps increment a counter.